src/runtime/prefetch.cpp

```diff
@@ -6,7 +6,9 @@ extern "C" {
 // parameter breaks a lot of optimizations, but needs to be WEAK
 // so that Codegen_LLVM can find an instance of the Function to insert.
 WEAK_INLINE int _halide_prefetch(const void *ptr) {
-    __builtin_prefetch(ptr, 1, 3);
+    constexpr int rw = 0;        // 1 = write, 0 = read
+    constexpr int locality = 0;  // 0 = no temporal locality, 3 = high temporal locality
```

Comment:
I wonder if we need to make these options parameters to the prefetch directive somehow. Experimenting with them does produce different results. However, I suspect these current values (read, no temporal locality) is probably the most useful/common case.